src/ToastContainer.js

```diff
@@ -288,18 +283,14 @@ class ToastContainer extends Component {
           {...this.getContainerProps(position, disablePointer)}
           key={`container-${position}`}
         >
-          {toastToRender[position].map(item => item)}
+          {toastToRender[position]}
```

Comment:
This will make the package break when using React <16. You probably should have bumped as a MAJOR change. 
3.0.0